src/collection-facets/more-facets-content.ts

```diff
@@ -0,0 +1,365 @@
+  }
+
+  private get renderMoreFacets() {
+    this.facetsLoading = false;
+    const min = (this.pageNumber - 1) * this.facetsPerPage;
+    const max = min + this.facetsPerPage - 1;
+
+    return html`<ul class="facet-list">
+      ${this.castedBuckets?.map((option, n) => {
+        const optionWrapperClass =
+          n >= min && n <= max ? 'farow' : 'farow hidden';
+
+        let displayText = option.key;
+        if (this.facetKey === 'language') {
+          displayText =
+            this.languageCodeHandler?.getLanguageNameFromCodeString(
+              displayText as string
+            ) ?? displayText;
+        }
+
+        return html` <li class=${optionWrapperClass}>
+          <div class="facet-row">
+            <label class="facet-info-display" title=${option.key}>
+              <input
+                type="checkbox"
+                class="selected-facets"
+                .value="${option.key}"
+                data-facet="${this.facetKey}"
+                @click=${(e: Event) => {
+                  this.facetClicked(e);
+                }}
+              />
+              <div class="facet-title">
+                ${this.facetKey !== 'collection'
+                  ? html`${displayText}`
+                  : html`<async-collection-name
+                      .collectionNameCache=${this.collectionNameCache}
+                      .identifier=${displayText}
+                      placeholder="-"
+                    ></async-collection-name>`}
+              </div>
+              <div class="facet-count">${option.doc_count}</div>
+            </label>
+          </div>
+        </li>`;
+      })}
+    </ul>`;
+  }
+
+  private facetClicked(e: Event) {
+    const { selectedFacets } = this;
+
+    const target = e.target as HTMLInputElement;
+    const { checked, value } = target;
+
+    let newFacets: SelectedFacets;
+    if (selectedFacets) {
+      newFacets = {
+        ...selectedFacets,
+      };
+    } else {
+      newFacets = defaultSelectedFacets;
+    }
+
+    if (checked) {
+      newFacets[this.facetKey as keyof typeof newFacets][value] = 'selected';
+    } else {
+      delete newFacets[this.facetKey as keyof typeof newFacets][value];
+    }
+
+    this.selectedFacets = newFacets;
+  }
+
+  private get loaderTemplate() {
+    return this.facetsLoading
+      ? html`<div class="facets-loader">
+          <ia-activity-indicator .mode="processing"></ia-activity-indicator>
+        </div>`
+      : '';
+  }
+
+  private get facetsPaginationTemplate() {
+    return html`
+      <more-facets-pagination
+        .paginationSize=${this.paginationSize}
+        .step=${Number(2)}
+        @pageNumberClicked=${this.pageNumberClicked}
+      ></more-facets-pagination>
+    `;
+  }
+
+  private get facetsContentTemplate() {
+    return html`
+      <div class="facets-content">${this.renderMoreFacets}</div>
+      ${this.paginationSize > 1 ? this.facetsPaginationTemplate : nothing}
+      <div class="footer">
+        <input
```

Comment:
can we make these buttons? I know inputs can turn into a button, that that feels like it's better to use that when there is a form...